testSelenium.py

```diff
@@ -0,0 +1,207 @@
+		'''
+			Purpose: To click each article (full text) within each issue, and extract the following information:
+				ID, Title, Abstract, Acknowledgements, Citation Authors, References, citationDOI
+
+			The function also opens a file for each new article it scrapes under the /jsonFilesAMS directory. 
+		'''
+
+		scrapeDriver = webdriver.PhantomJS()
+		
+		scrapeDriver.get(currLink)
+		scrapeDriver.find_element_by_xpath(xpath).click()
+		scrapeDriver.implicitly_wait(5)
+
+		seedUrl = scrapeDriver.current_url
+
+		self.get_seed_info(seedUrl)
+
+		json_dict = {}
+	 
+		if not os.path.exists(os.getcwd()+'/jsonFilesAMS'):
+			os.makedirs(os.getcwd()+'/jsonFilesAMS')
+
+		try:
+			URL = scrapeDriver.current_url
+			json_dict['id'] = URL
+		except:
+			print 'ID was not found on this page'
+			json_dict['id'] = 'Null'
+
+
+		try:
+			title = scrapeDriver.find_element_by_class_name('arttitle')
+			json_dict['title'] = title.text.encode('utf-8')
+			#print 'Title: ' + title.text.encode('utf-8')
+		except:
+			print 'Title was not found on this page'
+			json_dict['title'] = 'Null'
+
+
+		try:
+			abstract = scrapeDriver.find_element_by_class_name('abstractSection')
+			#print 'Abstract: ' + abstract.text.encode('utf-8')
+			json_dict['abstract'] = abstract.text.encode('utf-8')
+		except:
+			print 'Abstract was not found on this page'
+			json_dict['abstract'] = 'Null'
+
+
+		try:
+			ack = scrapeDriver.find_element_by_class_name('ack')
+			#print 'Acknowledgement: ' + ack.text.encode('utf-8')
+			json_dict['acknowledgement'] = ack.text.encode('utf-8')
+		except:
+			print 'Acknowledgements not found on this page'
+			json_dict['acknowledgement'] = 'Null'
+
+
+		try:
+			author = scrapeDriver.find_element_by_class_name('artAuthors')
+			#print 'Citation Authors: ' + artAuthors.text.encode('utf-8')
+			json_dict['citation_author'] = author.text.encode('utf-8')
+		except:
+			print 'Citation Authors not found on this page'
+			json_dict['citation_author'] = 'Null'
+
+
+		try:
+			references = scrapeDriver.find_element_by_class_name('references')
+			#print 'References: ' + references.text.encode('utf-8')
+			json_dict['article_references'] = references.text.encode('utf-8')
+		except:
+			print 'References not found on this page'
+			json_dict['article_references'] = 'Null'
+
+
+		try:
+			citationDOI = scrapeDriver.find_element_by_xpath('//*[@id="rightColumn"]/div[3]/a')
+			#print 'citationDOI: ' + citationDOI.text.encode('utf-8')
+			json_dict['citation_doi'] = citationDOI.text.encode('utf-8')
+		except:
+			print 'Citation DOI was not found on this page'
+			json_dict['citationDOI'] = 'Null'
+
+		filenameJSON = os.getcwd()+'/jsonFilesAMS/'+ json_dict['id'].split('://')[1].replace('/','-')+'.json'
+		with open(filenameJSON, 'w+') as f:
+			json.dump(json_dict, f)
+
+	def get_seed_info(self, currLink):
+		'''
+			Purpose: To compile a list of all the URLs explored in a consolidated text file. 
+		'''
+		file = open('seedList.txt', 'a');
```

Comment:
be careful of using special keywords for file names. e.g file
also when explicitly opening a file handler, remember to close it!